### PR TITLE
Display in zcharts the machine it is being tested on

### LINKED ISSUE #14: Display in zcharts the machine it is being tested on

### PR DESCRIPTION
**Pull request by lognorman20**
Closes #14

### CODE REVIEW

**Comment by mmaker** [comment]
omg thank you! this looks mostly ok -- just a few tiny changes for me to accept it.

**Comment by lognorman20** [comment]
Okay, anything in particular you want me to change?

**Comment by mmaker** [comment]
Check the code review!

**Comment by lognorman20** [comment]
Okay, how do I see the code review? I know about [this page](https://github.com/mmaker/zkalc/pull/35/files) but don't see any comments

**Comment by mmaker** [comment]
https://github.com/mmaker/zkalc/pull/35/files/8ae9dde8a09dea7b703d2ad7b90c60a5d36b7f72

**Comment by lognorman20** [comment]
Got it, thanks

File: frontend/app/charts/page.js [hunk 1]
```diff
@@ -120,6 +122,29 @@ const Home = () => {
...
+                  })
+                }
+                
+                return res;
+              }, [])
+            }}
+          >
+            <a onClick={(e) => e.preventDefault()}>
+              <Space>
+                {machines[machine].label}
```
[the remark this answers is not in the document]
Comment (lognorman20) [reply frontend/app/charts/page.js:142]: Updated 
image

**Comment by mmaker** [comment]
Great thanks!

**Pull request merged**